### DISCUSSION TITLE
[Connector]: Dremio

### DISCUSSION THREAD

**Discussion by a-wyatt** [evidence-dev/evidence#2432] [Data Source Request]
### What do you use this database for?

Primary analytics lakehouse. Provides a data mesh layer of data products that abstracts away complex relationships over a very large number of Parquet files stored on S3.

### Are you interested in contributing this connector?

I'm just making a suggestion

**Comment by archiewood** [comment]
I can't see any particularly popular NodeJS packages for Dremio, but this looks like a thin wrapper around the REST API which could be used fairly easily

https://github.com/christyharagan/dremio-node-api

**Comment by kenho811** [comment]
Thank you so much.

I deployed dremio at home and want to use evidence.dev to connect to dremio as well. 🙏

**Comment by ahirner** [comment]
How about using dremio over FlightSQL?
https://github.com/evidence-dev/evidence/discussions/2876
